PSIni/Public/Import-Ini.ps1

```diff
@@ -143,9 +146,11 @@ function Import-Ini {
                         $section = $script:NoSection
                         $ini[$section] = New-Object System.Collections.Specialized.OrderedDictionary([System.StringComparer]::OrdinalIgnoreCase)
                     }
-                    $name = $_
-                    Write-Verbose "$($MyInvocation.MyCommand.Name):: Adding key $name without a value"
-                    $ini[$section][$name] = $null
+                    $name = $_.Trim()
```

Comment:
If memory serves, the `.trim()` was removed as a feature request to keep spaces.

How do you feel about using the `.trim()` only for the `if` condition?
```pwsh
if (-not [string]::IsNullOrWhiteSpace($name.Trim())) {
```

Reply:
Umm.  
I have looked through the code and these two lines are interesting.  
With `.Trim()` followed by `IsNullOrwhiteSpace()`, we are essentially checking only for `IsNull()`.  

Trying to recall what I tried to achieve with this `Trim()`. I believe I tried to use it as a "fix" for malformed inis, with random prefix/sufix whitespaces, not matching expected strings (e.g. missing some " WHATEVER ", because I was looking for "WHATEVER")  

With this included, all a user has to be bothered with, is 
```PowerShell
$hashMap.ContainsKey(iniData[$section][$name])
```
without this, user has to remember, to always include a `Trim()` in their code.
```PowerShell
$hashMap.ContainsKey(iniData[$section][$name].Trim())
```

Reply:
So, it is your call @lipkau  
Do you want to preserve the prefix / suffix whitespace for some users with special / very rare needs?  
Or do we want to be nice to users without these special, very rare needs & fix their malformed INIs for them?

Reply:
I see this use case:
```
"    ooo`n  " > sample.ini
 $foo = Import-Ini sample.ini
 $foo["_"] 
```

I don't really see the benefit of stripping the whitespaces in such a key.
But we do it for the others too. I guess we should be consistent.

Reply:
Test added :)